app/styles/partials/rails.scss

```diff
@@ -0,0 +1,57 @@
+@media screen and (min-width : 1200px) {
+    .ui .rail.left {
+      left : auto;
+      right : 80%;
+      padding : 0 2rem 0 0;
+      margin : 0 2rem 0 0;
+    }
+    .rail.left + .event > .description {
```

Comment:
Again ... You're doing it only for the public event page. Don't use `.event > .description`. Keep it generic by referring to the div next to the rail div .